feat(serverless): precise public endpoint for edge (#6761)
scaleway/docs-content: pages/serverless-containers/how-to/deploy-container-behind-edge-services.mdx

This same procedure applies to **Serverless Functions**. The Edge Services backend stage supports both backends. Replace the container namespace and container resources with the equivalent Function namespace and function resources. Then use the corresponding function ID in the backend stage configuration.
</Message>

<Message type="important">

> **Edge Services** forwards incoming requests to your **Serverless Container** and **Serverless Functions** via its default (auto-generated) endpoint, not via a custom domain. The container used as an Edge Services backend must be public. **Edge Services** cannot reach a private or IAM-secured container.

Reviewer: ```suggestion
Edge Services forwards incoming requests to your Serverless Container and Serverless Functions through its default (auto-generated) endpoint, not through a custom domain. It cannot reach a private or IAM-secured container, so the backend container must be public.
```

<Message type="note">
    **Availability:** This feature is available via the API, CLI, SDK, and Terraform. It is **not available** in the Scaleway console.
</Message>

<Message type="note">

> You cannot currently disable the default endpoint of a **Serverless Container**  and a **Serverless Function** (for example, to keep only a custom domain). The default endpoint remains active. This capability planned for later. **Edge Services** currently relies on this default endpoint to forward requests to the container/function.

Reviewer 2: ```suggestion
You cannot currently disable the default endpoint of a **Serverless Container**  and a **Serverless Function** (for example, to keep only a custom domain). The default endpoint remains active. This capability is planned for later. **Edge Services** currently relies on this default endpoint to forward requests to the container/function.
```

> You cannot currently disable the default endpoint of a **Serverless Container**  and a **Serverless Function** (for example, to keep only a custom domain). The default endpoint remains active. This capability is planned for later. **Edge Services** currently relies on this default endpoint to forward requests to the container/function.

Reviewer: ```suggestion
You cannot disable the default endpoint of a Serverless Container or Serverless Function, for example to keep only a custom domain. This capability is planned for a future release. The default endpoint stays active, and Edge Services relies on it to forward requests to the container or function.
```
